pyproject.toml

```diff
@@ -1,35 +1,31 @@
-[tool.poetry]
+[project]
 name = "new-train-tracker"
 version = "2.4.1"
 description = "New Train Tracker by TransitMattters"
-authors = ["TransitMatters Labs Team"]
+authors = [{ name = "TransitMatters Labs Team" }]
+requires-python = ">=3.12,<3.13"
 license = "MIT"
-package-mode = false
+dependencies = [
+    "aiohttp~=3.12.13",
+    "boto3~=1.38.26",
+    "json-api-doc==0.15.0",
+    "pytz~=2025.2",
+    "schedule==1.2.2",
+    "datadog_lambda~=8.114.0",
+    "tzdata~=2025.2",
+]
```

Comment:
I like this less, the older format was much easier to read :(

Reply:
How so?

Reply:
Okay after loading it up locally, I'm less concerned. I was concerned these helpers would be missing in vscode and other ides
<img width="620" height="285" alt="Screenshot 2025-11-06 at 9 40 07 AM" src="https://github.com/user-attachments/assets/d70f5ae1-7eb9-4469-8949-04ba17e2f9fc" />


But looks like it works the same, so concern gone
<img width="618" height="195" alt="Screenshot 2025-11-06 at 9 40 24 AM" src="https://github.com/user-attachments/assets/da20c0da-7cc6-4228-b8b3-de68c4429903" />